packages/discord.js/typings/index.d.ts

```diff
@@ -3868,27 +3868,27 @@ export interface ApplicationCommandAutocompleteStringOptionData
   autocomplete: true;
 }
 
-export interface ApplicationCommandChoicesData extends Omit<BaseApplicationCommandOptionsData, 'autocomplete'> {
+export interface ApplicationCommandChoicesData<Type> extends Omit<BaseApplicationCommandOptionsData, 'autocomplete'> {
```

Comment:
```suggestion
export interface ApplicationCommandChoicesData<Type extends string | number = string | number> extends Omit<BaseApplicationCommandOptionsData, 'autocomplete'> {
```

without a default value, this is a breaking change for typings. Anyone using `ApplicationCommandChoicesData` in their code without `choice` option will get an error.